File: app/src/main/java/com/ubisys/uav/RetrofitConfig/SimpleApiCallback.java
package com.ubisys.uav.RetrofitConfig;



import retrofit2.Call;

/**
 * Created by liulinjie on 11/07/2016.
 * api回调封装
 */

public abstract class SimpleApiCallback<T> extends BaseApiCallback<T> {

    @Override
    protected void onApiError(Call<ApiResponse<T>> call, String code, String msg) {
//        ToastUtils.showShortToast(msg);

    }

    @Override
    public void onFailure(Call<ApiResponse<T>> call, Throwable t) {

    }

    @Override
    protected void onSuccess(Call<ApiResponse<T>> call, T t) {


    }

}

File: app/src/main/java/com/zzn/wanandroid/RetrofitConfig/SimpleApiCallback.kt
package com.zzn.wanandroid.RetrofitConfig


import retrofit2.Call

/**
 * Created by liulinjie on 11/07/2016.
 * api回调封装
 */

abstract class SimpleApiCallback<T> : BaseApiCallback<T>() {

    override fun onApiError(call: Call<ApiResponse<T>>, code: String?, msg: String?) {
        //        ToastUtils.showShortToast(msg);

    }

    override fun onFailure(call: Call<ApiResponse<T>>, t: Throwable) {

    }

    override fun onSuccess(call: Call<ApiResponse<T>>, t: T?) {


    }

}
